src/protagonist/DLCS.Hydra.Tests/Model/Converters/ImageDeliveryChannelsConverterTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿using DLCS.HydraModel;
+using FluentAssertions;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace DLCS.Hydra.Tests.Model.Converters;
+
+public class ImageDeliveryChannelsConverterTests
+{
+    [Fact]
+    public void DeliveryChannelsConverter_Supports_Basic_Channels()
+    {
+        var hydraAssetBody = @"{
+            ""@type"": ""Image"",
+            ""origin"": ""https://example.org/asset.tiff"",
+            ""family"": ""I"",
+            ""mediaType"": ""image/tiff"",
+            ""deliveryChannels"": [""iiif-img"",""thumbs"",""file""]
```

Comment:
assume it works, but it might be worth having a test for `"deliveryChannels: null"`